waspc/README.md

```diff
@@ -156,10 +156,10 @@ When done, new tab in your browser should open and you will see a Todo App!
    - NOTE: You will need to install `ghcid` globally first. You can do it with `cabal install ghcid`.
 3. Do a change in the codebase (most often in `src/` or `cli/src/` or `data/`), and update tests if that makes sense (see [Test](#tests)).
    Fix any errors shown by HLS/`ghcid`.
-   Rinse and repeat. If you're an internal team member, postpone updating e2e tests until approval (see [here](#note-for-team-members)).
+   Rinse and repeat. If you're an internal team member, postpone updating snapshot tests until approval (see [here](#note-for-team-members)).
 4. Use `./run build` to build the Haskell/cabal project, and `./run wasp-cli` to both build and run it. If you changed code in `packages/`, you will also need to run `./run build:packages` (check [TypeScript Packages section](#typescript-packages) for more details). Alternatively, you can also run slower `./run build:all` to at the same time build Haskell, TS packages, and any other piece of the project in one command.
 5. For easier manual testing of the changes you did on a Wasp app, you have the `examples/todoApp` app, which we always keep updated. Also, if you added a new feature, add it to this app (+ tests) if needed. Check its README for more details (including how to run it).
-6. Run `./run test` to confirm that all the tests are passing (unit + integration + e2e). If needed, accept changes in the e2e golden/snapshot tests with `./run test:e2e:accept-all`. Check "Tests" for more info.
+6. Run `./run test` to confirm that all the tests are passing (`waspc` + Wasp CLI + Wasp LS + `todoApp` + examples + starters). If needed, accept changes in the snapshot tests with `./run test:waspc:e2e:accept-all`. Check "Tests" for more info.
```

Comment:
If we decided to rename e2e tests into snapshot tests, we should apply that change everywhere.

This means that:
- The command `test:waspc:e2e:accept-all` should become something like `test:waspc:snapshots:accept-all`.
- The code should reflect those changes (in `run.sh` and anywhere else).
- The file system names should reflect those changes too.

The rule of thumb is this: If a new person started working on Wasp, they should never see any indication that snapshot tests were ever called e2e tests.

Reply:
This is another place where we list all the tests. I think we can kick out everything in the parentheses. It's not that useful and can easily become outdated and confusing.